dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'twitter_ads_source'
-version: '0.4.0'
+version: '0.4.1'
```

Comment:
Would you be able to make this same update within the `integration_tests/dbt_project.yml` file as well.

Reply:
Updated the version in [033d6b1](https://github.com/fivetran/dbt_twitter_source/pull/15/commits/033d6b1a43f948de266cbd191f24f2b4584ff78d)